examples/cxx/complex_data_types/BUILD.bazel

```diff
@@ -18,7 +18,7 @@ cc_binary(
         "src/complex_data_types.cpp",
     ],
     deps = [
-        "@iceoryx//:iceoryx_hoofs",
+        "//iceoryx2-bb/cxx:iceoryx2-bb-cxx",
```

Comment:
With an alias in the root BUILD.bazel the dependency could be called like the others:

```
alias(
    name = "iceoryx2-bb-cxx",
    actual = "//iceoryx2-bb/cxx:iceoryx2-bb-cxx",
    visibility = ["//visibility:public"],
)
```

We can do it afterwards as well.

Reply:
I'm working already on a follow up PR to make iceoryx2 and iceoryx classic co-installable. This can be done in that PR.